ssdutil/main.py

```diff
@@ -65,6 +75,11 @@ def ssdutil():
     parser.add_argument("-e", "--vendor", action="store_true", default=False, help="Show vendor output (extended output if provided by platform vendor)")
     args = parser.parse_args()
 
+    disk_type = get_disk_type(args.device)
+    if disk_type != DISK_TYPE_SSD:
+        print("Disk is not SSD")
+        sys.exit(1)
```

Comment:
@sujinmkang , can you also ensure that ssd related testcases in sonic-mgmt are skipped for such platforms after this change gets merged?

Reply:
Azure/sonic-mgmt#4807